CHANGELOG.md

```diff
@@ -6,6 +6,153 @@
 
 Releases are now named after famous [Cricket](https://en.wikipedia.org/wiki/Cricket) players.
 
+## [0.5] - [TBD] - 2017-11-??
+
+JupyterHub 0.8, HTTPS & scalability.
+
+### Upgrading from 0.4
+
+#### DB upgrade
+
+This release contains a big JupyterHub version bump (from 0.7.2 to 0.8). If
+you are using the default database provider (sqlite), then the required db upgrades
+will be performed automatically when you do a `helm upgrade`.
+
+If you are using MySQL / PostgreSQL as your backing database, you will need to
+perform a manual database upgrade with the following steps:
+
+1. Make a full backup of your database, just in case things go bad.
+2. Make sure that the database user used by JupyterHub to connect to your database
+   can perform schema migrations like adding new tables, altering tables, etc.
+3. In your config.yaml, add the following config:
+
+   ```yaml
+   hub:
+     db:
+       upgrade: true
+   ```
+4. Do a `helm upgrade`. This should perform the database upgrade needed.
+5. Remove the lines you added in step 3, and do another `helm upgrade`.
```

Comment:
Remove the lines added in step 3. Now, to update the jupyterhub run another `helm upgrade`.